bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+ABDRUCK
+ABEND
+ABFALL
+ABFLUG
+ABFOLGE
+ABGAS
+ABGLEICH
+ABHANG
+ABITUR
+ABKOMMEN
+ABMARSCH
+ABONNENT
+ABREISE
+ABSCHIED
+ABSEITS
+ABSICHT
+ABSPANN
+ABSTRICH
+ABTEIL
+ABWASCH
+ABZWEIG
+ACHSEL
+ACHTUNG
+ACKERBAU
+ADAPTER
+ADDITION
+ADJEKTIV
+ADJUDANT
+ADLER
+ADOPTION
+AFFE
+AFRIKA
+AGENDA
+AHNUNG
+AHORN
+AKADEMIE
+AKAZIE
+AKKORD
+AKTE
+AKTION
+AKUSTIK
+AKZENT
+ALCHEMIE
+ALIMENTE
+ALLEE
+ALLIANZ
+ALLMACHT
+ALLRAD
+ALLTAG
+ALPAKA
+ALPEN
+ALPHABET
+ALTERTUM
+ALTSTADT
+ALTWAGEN
+ALUFOLIE
+AMBITION
+AMBOSS
+AMBULANZ
+AMERIKA
+AMME
+AMMONIAK
+AMNESTIE
+AMPEL
+AMPULLE
+AMTSEID
+AMULETT
+ANAGRAMM
+ANAKONDA
+ANALOGIE
+ANBIETER
+ANDENKEN
+ANEKDOTE
+ANFAHRT
+ANFRAGE
+ANGABE
+ANGEBOT
+ANGRIFF
+ANIMATOR
+ANIS
+ANKER
+ANKUNFT
+ANLAGE
+ANLEIHE
+ANLIEGER
+ANMELDER
+ANMUTUNG
+ANNAHME
+ANNEXION
+ANODE
+ANORAK
+ANPROBE
+ANREDE
+ANRUF
+ANSATZ
+ANSCHEIN
+ANSEHEN
+ANSPORN
+ANSTAND
+ANTIKE
+ANTRAG
+ANTWORT
+ANWALT
+ANWENDER
+ANWOHNER
+ANZAHL
+ANZEIGE
+ANZUG
+AORTA
+APERITIF
+APFEL
+APOTHEKE
+APPARAT
+APPELL
+APPLAUS
+APRIKOSE
+AQUARELL
+ARA
+ARBEIT
+ARCHIV
+AREAL
+ARGON
+ARGUMENT
+ARIE
+ARKADE
+ARMATUR
+ARMBAND
+ARMEE
+ARMREIF
+ARMUT
+ARNIKA
+ARROGANZ
+ARSEN
+ARTISTIK
+ARZNEI
+ARZT
+ASIEN
+ASPEKT
+ASPIRANT
+ASTEROID
+ASTGABEL
+ASYL
+ATEMZUG
+ATHLET
+ATMUNG
+ATOLL
+ATOMUHR
+ATTACKE
+ATTEST
+ATTRAPPE
+AUERHAHN
+AUFBRUCH
+AUFDRUCK
+AUFGUSS
+AUFLAGE
+AUFPRALL
+AUFRUF
+AUFSATZ
+AUFTRAG
+AUFWAND
+AUFZUG
+AUGAPFEL
+AUSBLICK
+AUSDAUER
+AUSFLUG
+AUSGABE
+AUSHANG
+AUSKUNFT
+AUSLESE
+AUSNAHME
+AUSPUFF
+AUSREDE
+AUSSAGE
+AUSTER
+AUSWEIS
+AUSZEIT
+AUTOMAT
+AVOKADO
+AXIOM
+AXT
+BACHLAUF
+BAD
+BAGGER
+BAHNHOF
+BAHRE
+BALKEN
+BALLON
+BALSAM
+BALZ
+BANANE
+BANDIT
+BANKETT
+BARACKE
+BARCODE
+BARGELD
+BAROCK
+BARREN
+BARSCH
+BASAR
+BASIS
+BASSIST
+BATIK
+BAUCHWEH
+BAUER
+BAUGRUBE
+BAUHAUS
+BAUKLOTZ
+BAULAND
+BAUMARKT
+BAUPLAN
+BAUSATZ
+BAUTEIL
+BAUWERK
+BEAMTER
+BEBEN
+BECHER
+BECKEN
+BEDARF
+BEDIENER
+BEERE
+BEET
+BEFEHL
+BEFUND
+BEGABUNG
+BEGINN
+BEICHTE
+BEIFALL
+BEIGABE
+BEIKOST
+BEILAGE
+BEISPIEL
+BEITRAG
+BEIWERK
+BELANGE
+BELEG
+BELIEBEN
+BEMALUNG
+BENEHMEN
+BENGEL
+BENOTUNG
+BENUTZER
+BENZIN
+BERATER
+BEREICH
+BERG
+BERICHT
+BERUF
+BESATZER
+BESCHEID
+BESITZ
+BESTECK
+BESUCH
+BETONUNG
+BETRIEB
+BETTRUHE
+BEUTEL
+BEWEIS
+BEWOHNER
+BEZIRK
+BEZUG
+BIATHLON
+BIEGUNG
+BIER
+BIEST
+BIKINI
+BILANZ
+BILD
+BINDUNG
+BIOGAS
+BIOLOGE
+BIOMASSE
+BIOTOP
+BIRKHAHN
+BIRNE
+BISTRO
+BITTE
+BITUMEN
+BIWAK
+BIZEPS
+BLASE
+BLATT
+BLAUWAL
+BLEICHE
+BLESSUR
+BLICK
+BLINKER
+BLITZ
+BLOCKADE
+BLUME
+BODEN
+BOHNE
+BOHRER
+BOILER
+BOJE
+BOLLWERK
+BOLZEN
+BOMBE
+BONSAI
+BOOT
+BORD
+BORSTE
+BOSHEIT
+BOTTICH
+BOYKOTT
+BRANDUNG
+BRATEN
+BRAUEREI
+BREMSE
+BRENNGUT
+BRETT
+BREZE
```

Comment:
```suggestion
BREZEL
```
![image](https://user-images.githubusercontent.com/1100176/108707857-4d831300-7519-11eb-9d09-d54e25f272ba.png)

Different word variants exist, I think it would be fine to pick "Brezel", that's how Wikipedia lists it: https://de.wikipedia.org/wiki/Brezel

Reply:
You prefer to keep the word written as "Brezel" or to replace it completely?

Reply:
Word replaced